scripts/vscripts/tf2ware_ultimate/minigames/pick_up_that_can.nut

```diff
@@ -0,0 +1,222 @@
+minigame <- Ware_MinigameData
+({
+	name          = "Pick Up That Can"
+	author		  = "CzechMate"
+	description   = "Pick Up And Trash The Can!"
+	duration      = 7.0
+	music         = "sweetdays"
+})
+
+::trashcan_model <- "models/props_trainstation/trashcan_indoor001b.mdl"
+::can_model <- "models/props_junk/popcan01a.mdl"
+
+////////////////////////// --- Settings --- ///////////////////////////////////////////
+
+const PickupDistance = 100.0; // Distance the player can pick up the prop from

```

Comment:
Do not use "const" in minigames as it goes into global scope. Just declare these as normal variables, i.e. local PickupDistance = 100